### PR TITLE
feat(dh): license header 

### PR DESCRIPTION
**Pull request by dzhavat**
### Workspace

- Set year in license header templates to 2020
- Update year in license header for all existing files

Note: New files will get license headers with current year while existing files will retain their current license headers.

- https://github.com/Energinet-DataHub/geh-metering-point/issues/624

### CODE REVIEW

**Review by LayZeeDK — DISMISSED** [review]

**Review by frontend-specialisten — APPROVED** [review]

**Comment by dzhavat** [comment]
**Block**: Waiting for clarification whether the year should be updated or not.

**Comment by dzhavat** [comment]
**Unblocked**: The year in the license header should be set to 2020 for all relevant files, new and existing. 2020 because that is the year when DataHub was initially open-sourced. I will make a follow-up commit with the necessary adjustments.

**Comment by dzhavat** [comment]
@LayZeeDK Heads up that the changes in this PR have an effect on the files for EnergyOrigin.

File: .github/actions/publish-release-contents/action.yml [hunk 1]
```diff
@@ -32,4 +32,4 @@ runs:
       uses: actions/download-artifact@v2
       with:
         name: bff
-        path: ${{ inputs.RELEASE_FOLDER_PATH }}/functions/bff
\ No newline at end of file
+        path: ${{ inputs.RELEASE_FOLDER_PATH }}/functions/bff
```
Comment (frontend-specialisten) [inline .github/actions/publish-release-contents/action.yml:35]: Question: Is this change intended?

**Review by frontend-specialisten — CHANGES_REQUESTED** [review]

File: .github/actions/publish-release-contents/action.yml (hunk 1 above)
Comment (dzhavat) [reply .github/actions/publish-release-contents/action.yml:35]: I don't know why it appears. It wasn't intended at least. Must be a change only git cam see.

**Review by frontend-specialisten — APPROVED** [review]

**Review by frontend-specialisten — APPROVED** [review]
